app-web/src/components/Form/CurrentForm.js

```diff
@@ -0,0 +1,93 @@
+import { React, useMemo, useState, Fragment } from 'react';
+import { SelectDropdown } from './form';
+import { graphql, useStaticQuery } from 'gatsby';
+import { flattenGatsbyGraphQL } from '../../utils/dataHelpers';
+
+export const CurrentForm = () => {
+  const [currVal, setVal] = useState('');
+  let content;
+
+  if (currVal === 'addToTopic') {
+    content = (
+      <Fragment>
+        <DataQuery datatype="topics"></DataQuery>
+        <SelectDropdown>
+          <option value="">Select sourcetype</option>
+          <option value="web">Web</option>
+          <option value="github">Github</option>
+        </SelectDropdown>
+      </Fragment>
+    );
+  } else if (currVal === 'addToJourney') {
+    content = <DataQuery datatype="journeys"></DataQuery>;
+  }
+
+  return (
+    <Fragment>
+      <form>
+        <SelectDropdown
+          label="Would you like to add to an existing Topic or Journey"
+          name="jobType"
+          onChange={e => {
+            setVal(e.target.value);
+          }}
+        >
+          <option value="">Select a job type</option>
+          <option value="addToTopic">Topic</option>
+          <option value="addToJourney">Journey</option>
+        </SelectDropdown>
+      </form>
+      {content}
+    </Fragment>
+  );
+};
+
+export default CurrentForm;
+
+const DataQuery = ({ datatype }) => {
```

Comment:
whats this for?

Reply:
Haven’t completely build out this part of the form yet. But this is for adding content to existing resources (topics and journeys). Used query to import topic and journey names, and put them into respective dropdowns